### PR TITLE
Infra wg notes

### PR DESCRIPTION
**Pull request by nibalizer**
Notes from 10-08-2020 meeting

### CODE REVIEW

File: meeting-notes/2020-10-08-infra-wg.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+- Spencer Krum
+
+### Who Joined
+
+- Leonardo Grasso ([leogr](https://github.com/leogr))
+- Umang Vanjara ([vanjara](https://github.com/vanjara))
+- Jonah Jones ([jonahjon](https://github.com/jonahjon))
+- Spencer Krum ([nibz](https://github.com/nibalizer))
+
```
Comment (fntlnz) [inline meeting-notes/2020-10-08-infra-wg.md:16]:
```suggestion
- Lorenzo Fontana ([fntlnz](https://github.com/fntlnz))
```

**Review by leogr — APPROVED** [review]

File: meeting-notes/2020-10-08-infra-wg.md (hunk 1 above)
Comment (maxgio92) [reply meeting-notes/2020-10-08-infra-wg.md]: Can I make a review of a review to add my name on the 17th line? :D

Comment (leogr) [reply meeting-notes/2020-10-08-infra-wg.md]: Hey @maxgio92 

sorry, I did not notice that, could you fill another PR for that, please?

Comment (maxgio92) [reply meeting-notes/2020-10-08-infra-wg.md]: Sure @leogr! Done

**Pull request merged**